source/connect/connections.txt

```diff
@@ -0,0 +1,85 @@
+.. _connections-tab:
+
+===================
+Connections Sidebar
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The :guilabel:`Connections Sidebar` shows MongoDB deployments that
+you can interact with through MongoDB Compass. 
+
+.. note::
+
+   .. include:: /includes/multiple-connections.rst
+
+Connections Sidebar
+-------------------
+
+The :guilabel:`Connections` sidebar on the left of the Compass window
```

Comment:
[question] Is it :guilabel:`Connections Sidebar` or :guilabel:`Connections` sidebar? We want to make sure we're consistent here!